### DISCUSSION TITLE
configure with nginx

### DISCUSSION THREAD

**Discussion by amirofski** [postalserver/postal#1638] [Help with using Postal] [answered]
Hello . 
i have a site on vps with nginx server . i install postal 2 but it's not working and when i use "`postal status`" return postal_worker_1 exit error

**Comment by willpower232** [comment, marked as answer]
I don't think this is an nginx error, you should try `postal logs worker` to see what its problem is

**Reply by amirofski** [reply]
Thank you, fixed this Problem, But now caddy can't working :))

**Reply by willpower232** [reply]
What error message do you have?

Also you should not be running both nginx and caddy as they will likely conflict with each other.
